oggm/graphics.py

```diff
@@ -390,7 +390,7 @@ def plot_distributed_thickness(gdir, ax=None, cleomap=None, how=None):
             cleomap.set_geometry(l, crs=crs, color='black', linewidth=0.5)
 
     cleomap.set_cmap(plt.get_cmap('viridis'))
-    cleomap.set_plot_params(nlevels=256)
+    cleomap.set_plot_params(nlevels=256, vmax=1100)
     cleomap.set_data(thick)
```

Comment:
For this PR it would be good to remove this, but I will make the plotting functions more flexible in a separate PR: you should be able to control plotting levels from outside the function